applications/openshift/networking/routes_rate_limit/rule.yml

```diff
@@ -3,8 +3,33 @@ title: Ensure that all Routes has rate limit enabled
 
 description: |-
   OpenShift has an option to set the rate limit for Routes [1] when creating new Routes.
-  All routes outside the openshift namespaces and the kube namespaces should use the
-  rate-limiting annotations.
+  All routes outside the excluded namespaces (as defined by the variable
+  <tt>var-routes-excluded-namespaces-regex</tt>) should use the rate-limiting annotations.
+  
+  In case additional namespaces need to be excluded from this check (e.g., operator-managed
+  namespaces where direct route modifications are not recommended), create a
+  <tt>TailoredProfile</tt> and add the additional namespaces to the regular expression
+  in the variable <tt>var-routes-excluded-namespaces-regex</tt>.
+  An example allowing additional namespaces follows:
+  <pre>
+  apiVersion: compliance.openshift.io/v1alpha1
+  kind: TailoredProfile
```

Comment:
We don't typically put references like this in the rule descriptions. A reference to the documentation might be fine. Also, users consuming these through ACS might have a disjointed experience unless they can access the clusters this applies to.